VirtualFilesystemTestTrait.php

```diff
@@ -139,4 +148,25 @@ public function getDefaultVfs() {
 			'wp-config.php' => '',
 		];
 	}
+
+	/**
+	 * Initializes the original files and directories properties for use in the tests.
+	 */
+	protected function initOriginals() {
```

Comment:
@Screenfeed Take a look at this new method. I combined what you were doing plus added a way to skip running this method. What do you think? Does this achieve the intent and needs you wanted in this PR?

Reply:
@hellofromtonya LOL. Much better than what I did ^^' (`$structure = $this->config['structure'];`)